tests/unit/test_finder.py

```diff
@@ -502,7 +505,7 @@ def test_finder_installs_pre_releases_with_version_spec():
         assert link.url == "https://foo/bar-2.0b1.tar.gz"
 
 
-class test_link_package_versions(object):
+class TestLinkPackageVersions(object):
```

Comment:
Pytest is quite strict on how tests in a class are picked up.

Reply:
Yep. I think that's a good thing.

This is a configurable default but I prefer camel case for class names because PEP 8.